components/dialogs/ResourceUploadDialog.tsx

```
@@ -177,50 +159,91 @@ export function ResourceUploadDialog({
+        );
+        const failedUploads = uploadResults.filter(
+          (result) => result.status === "rejected"
+        );
+
+        if (failedUploads.length > 0) {
+          const errorMessages = failedUploads
+            .map(
+              (result) => result.reason.response?.data?.error || "Unknown error"
+            )
+            .join("\n");
+          toast.error("Failed to upload resources:\n" + errorMessages);
```

Biome (1.9.4)

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)